.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Can/should this specified with a --flag (or disabled if `black` already checks this)?
If we do we can get rid of this file.

Reply:
Fair enough.